draft-ietf-quic-transport.md

```diff
@@ -1266,6 +1266,42 @@ actual exponent is one-less than the explicit exponent, and the value represents
 4096 microseconds.  Any values larger than the representable range are clamped
 to 0xFFFF.
 
+
+### ACK Frames and Packet Protection
+
+ACK frames that acknowledge protected packets MUST be carried in a packet that
+has an equivalent level of packet protection.  Functionally, this means:
+
+* Unprotected packets, such as those that carry the initial cryptographic
+  handshake messages, can be acknowledged in unprotected packets.  This has
```

Comment:
I don't understand why this text says they "can" be acked in unprotected packets. Shouldn't we require that they MUST?